### ISSUE TITLE
Beta release - Package: setuptools

### ISSUE THREAD

**Issue by fnoir** [BadWolf42/jMQTT#91]
Can you add "python3-setuptools" package on step "Install dependancies" ?

Error during installation:

```
*
* Install python3 paho-mqtt library
*
Collecting paho-mqtt
  Using cached https://files.pythonhosted.org/packages/32/d3/6dcb8fd14746fcde6a556f932b5de8bea8fedcb85b3a092e0e986372c0e7/paho-mqtt-1.5.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ModuleNotFoundError: No module named 'setuptools'
```

I use a minimal Debian 10 for Jeedom.

**Comment by BadWolf42** [comment]
Hi @fnoir, this should be fixed as per PR #95.
We decided to install it using pip3 instead of apt. No issue though, pip3 does not update an apt sourced package.
If you have any chance to test otherwise will close this issue.

**Comment by fnoir** [comment]
Hi @BadWolf42,
I tested and it works (i removed "python3-setuptools" package before)!

```
*
* Install python3 setuptools library
*
Collecting setuptools
  Downloading https://files.pythonhosted.org/packages/37/72/18182298997d20e333d4bae0bead740add1b09ca99200eae65a73d9aa732/setuptools-58.0.3-py3-none-any.whl (816kB)
Installing collected packages: setuptools
Successfully installed setuptools-58.0.3
*
* Install python3 paho-mqtt library
*
Requirement already up-to-date: paho-mqtt in /usr/local/lib/python3.7/dist-packages (1.5.1)
```
